src/scripts/settings/userSettings.js

```diff
@@ -429,6 +429,19 @@ export class UserSettings {
         return this.get('screensaver', false);
     }
 
+    /**
+     * Get or set the interval between backdrops when using the backdrop screensaver.
+     * @param {number|undefined} val - The interval between backdrops in milliseconds.
+     * @return {number} The interval between backdrops in milliseconds.
```

Comment:
The 1000 multiplier you mentioned is actually because swiper expects the delay milliseconds. Alternatively it would be possible to let the user enter it in milliseconds and remove the multiplier but I think that would degrade the user experience.